Editor/CodeAnalysis/CallTreeNode.cs

```diff
@@ -6,15 +6,15 @@
 
 namespace Unity.ProjectAuditor.Editor.CodeAnalysis
 {
-    public class CallTreeNode : DependencyNode
+    internal class CallTreeNode : DependencyNode
     {
-        internal readonly string m_Name;
+        public readonly string m_Name;
```

Comment:
So this... is complicated. I spoke to Marco about it. The consensus we arrived at was that everything that could possible be marked as internal probably should be (because it would make it easier to spot things that have been made public in future and check that the change is accompanied by proper documentation), but that in the interests of getting the package promoted in a reasonable time-frame, we could gloss over it for this pull request and revisit it in the future.

Reply:
> So this... is complicated. I spoke to Marco about it. The consensus we arrived at was that everything that could possible be marked as internal probably should be (because it would make it easier to spot things that have been made public in future and check that the change is accompanied by proper documentation), but that in the interests of getting the package promoted in a reasonable time-frame, we could gloss over it for this pull request and revisit it in the future.

Sounds good, so far I could also only spot rather nit-picking "issues".